model/http-common.yaml

```
@@ -62,6 +62,25 @@ groups:
           conditionally_required: If and only if one was received/sent.
         brief: '[HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).'
         examples: [200]
+      - ref: error.id
+        requirement_level:
+          conditionally_required: If request has ended with an error.
+        examples: ['timeout', 'name_resolution_error', 'server_certificate_invalid']
+        note: |
+          If response status code was sent or received and status indicates an error according to [HTTP span status definition](/docs/http/http-spans.md),
+          `error.id` SHOULD be set to the canonical [Reason Phrase](https://www.rfc-editor.org/rfc/rfc2616.html#section-6.1.1)
+          corresponding to the returned status code or to `_OTHER`.
+
+          If request fails with an error before or after status code was sent or received, such error SHOULD be
+          recorded on `error.id` attribute. The description SHOULD be predictable and SHOULD have low cardinality.
```

```suggestion
          recorded in the `error.id` attribute. The value used for `error.id` SHOULD be predictable and SHOULD have low cardinality.
```
I think it's best to avoid "description" for `error.id` otherwise people might be confused.